### PR TITLE
tests: report probable serialization errors

### PR DESCRIPTION
**Pull request by jirikuncar**
Makes sure that serialization errors are reported early.

### CODE REVIEW

File: tests/api/v1/datadog/features/synthetics.feature [hunk 1]
```diff
@@ -277,7 +277,6 @@ Feature: Synthetics
     When the request is sent
     Then the response status is 200 OK
 
-  @generated @skip
```
Comment (jirikuncar) [inline tests/api/v1/datadog/features/synthetics.feature:280]: 🔙

**Review by zippolyte — DISMISSED** [review]

File: tests/api/v1/datadog/features/synthetics.feature [hunk 2]
```diff
@@ -277,7 +277,6 @@ Feature: Synthetics
...
   Scenario: Get the list of all tests returns "OK - Returns the list of all Synthetic tests." response
```
Comment (jirikuncar) [inline tests/api/v1/datadog/features/synthetics.feature:280]:
```suggestion
  @generated @skip
  Scenario: Get the list of all tests returns "OK - Returns the list of all Synthetic tests." response
```

**Review by zippolyte — APPROVED** [review]

**Pull request merged**